sqlglot/dialects/clickhouse.py

```diff
@@ -489,6 +654,15 @@ def parameterizedagg_sql(self, expression: exp.Anonymous) -> str:
             params = self.expressions(expression, key="params", flat=True)
             return self.func(expression.name, *expression.expressions) + f"({params})"
 
+        def anonymousaggfunc_sql(self, expression: exp.Anonymous) -> str:
+            return self.func(expression.name, *expression.expressions)
+
+        def combinedaggfunc_sql(self, expression: exp.Anonymous) -> str:
+            return self.anonymousaggfunc_sql(expression)
+
+        def combinedparameterizedagg_sql(self, expression: exp.Anonymous) -> str:
```

Comment:
are the types here right? shouldn't it be like CombinedParameterizedAgg ?

Reply:
It works, but yeah, expression probably needs to be `exp.CombinedParameterizedAgg`, lol